client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -219,6 +272,16 @@ export const useCommandsArrayWpcom = ( {
 		{
 			name: 'manageDomains',
 			label: __( 'Manage domains' ),
+			searchLabel: [
+				_x( 'dns', 'Keyword for the Manage domains command' ),
```

Comment:
Should we add "Manage domains" as a keyword too, to let the user search for this command by its display label?